src/abstractions/serialization/KiotaJsonSerializer.Serialization.cs

```diff
@@ -20,9 +20,21 @@ public static partial class KiotaJsonSerializer
     /// </summary>
     /// <typeparam name="T">Type of the object to serialize</typeparam>
     /// <param name="value">The object to serialize.</param>
+    /// <param name="useBackingStore">Backing store keeps track of changes, setting this to false will give you the full item.</param>
```

Comment:
could we name this parameter something else like "serializeOnlyChangedValues" ?

Reply:
Yes